### PR TITLE
doc:BGCC Week 9 feedback

### PR DESCRIPTION
**Pull request by nicoloboo02**
# Description
Week 8 feedback has been added
# Checklist:
- [ ] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [ ] I have checked that there is no fragile information published unintentionally.

### CODE REVIEW

File: docs/Anexo I. Aportación a la BGCC/Semana 8.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## (19/03/2024 - 25/03/2024)
+## Semana 8
+
+### Aspectos importantes para la presentación:
+
+- Descripción clara antes de la demostración Es esencial proporcionar una explicación precisa de lo que se mostrará durante la demo.
+
+- Usabilidad de la demo Se recomienda evitar un exceso de marketing al inicio y utilizar el zoom de manera moderada para no marear al espectador.
+
+- Feedback recibido y gestión del mismo\*\*: Se debe incluir una sección que detalle el feedback recibido y cómo se está gestionando, así como una lista priorizada por relevancia.
```
Comment (alvarobernal2412) [inline docs/Anexo I. Aportación a la BGCC/Semana 8.md:10]: \*\* What is this?

File: docs/Anexo I. Aportación a la BGCC/Semana 8.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Inclusión de usuarios piloto y planificación Falta mostrar la parte de usuarios piloto en la presentación, así como la planificación del proyecto.
+
+- Storytelling en el storyboard Se sugiere que el storyboard cuente una historia coherente para mejorar la comprensión del producto.
```
Comment (alvarobernal2412) [inline docs/Anexo I. Aportación a la BGCC/Semana 8.md:14]: Quite redundant

**Review by alvarobernal2412 — CHANGES_REQUESTED** [review]
The rest of the sections are very good

**Comment by nicoloboo02** [comment]
I deleted it so there is no problem with the content.

**Review by alvarobernal2412 — APPROVED** [review]
Thanks!!

**Pull request merged**